### ISSUE TITLE
Beet hackathon demo - Beet if reopened creates new process ids and the old process ids stay the same (Observed on Linux may be windows users wont face this issue) I needed to manually kill them for the app to work.

### ISSUE THREAD

**Issue by manikey123** [bitshares/beet#124]
Please assign to me @sschiessl-bcp

**Comment by sschiessl-bcp** [comment]
Can you describe the steps to reproduce?

**Comment by manikey123** [comment]
Below  lines are spawning redundant threads. Resulting in stale UI. That needs to be manually killed.
Do suggest recommendation for the below lines. Should they be removed. Then the connection icon wont come. Plz suggest any ideas
@sschiessl-bcp 

![image](https://user-images.githubusercontent.com/22608398/58534158-56f21680-819f-11e9-8da2-566a9d98f00d.png)

![image](https://user-images.githubusercontent.com/22608398/58534031-ffec4180-819e-11e9-8881-25085b8fd83c.png)

**Comment by sschiessl-bcp** [comment]
How to reproduce?

**Comment by manikey123** [comment]
Use Linux OS and then start the beet app.
Observed that the line keeps frequently repeating.

**Comment by manikey123** [comment]
In order to fix above issue I researched AppImg
https://appimage.org/
AppImg is open source. Set ups are handled in the AppImg itself and it would be seamless for Linux users

AppImg is also implemented by Scatter
https://get-scatter.com/download

This also helps solve below setups which the user needs to do manually
https://github.com/bitshares/beet/issues/123#issuecomment-496310678

**Comment by sschiessl-bcp** [comment]
Yes, ideally we would be building AppImage through travis. Please continue
